aptoide-ui/src/main/java/cm/aptoide/pt/aptoide_ui/theme/Theme.kt

```diff
@@ -81,3 +42,16 @@ fun AptoideTheme(
     )
   }
 }
+
+fun getBaseAppTheme(): String {
+  return BuildConfig.APTOIDE_THEME
+}
+
+private fun getThemeColors(baseTheme: String, isDarkTheme: Boolean): AppColors {
+  return if (baseTheme == "cobrand") {
```

Comment:
can't we use always the same config file but branched in different folders (vanilla & cobrand)?